src/renderers/native/ReactNativeMount.js

```diff
@@ -175,7 +175,7 @@ var ReactNativeMount = {
    * @param {View} view View tree image.
    * @param {number} containerViewID View to insert sub-view into.
    */
-  _mountImageIntoNode: function(mountImage, containerID) {
+  _mountImageIntoNode: function(mountImage : number, containerID : number) {
```

Comment:
This is a private method which shouldn't really need any annotations. Not sure if this is a particular config we need to have.